### DISCUSSION TITLE
Including a file to an already included file question

### DISCUSSION THREAD

**Discussion by piotrwalczak1** [serverless/serverless#11103] [General]
Can you please take a look at this ?
https://stackoverflow.com/questions/72323846/is-there-a-possibility-to-include-a-file-that-is-already-included

I can't find a way how to fix this problem.

**Comment by pgrzesik** [comment]
Hello @piotrwalczak1 - I've read the SO issue and I'm having a hard time fully understanding the problem. It is possible to override roles for specific functions, but it's not possible to assign more than one role - it's just a limitation of AWS - each Lambda function can have only 1 role assigned. 

Could you please clarify what exactly are you trying to achieve? Why do you want to assign more than one role to a function?

**Reply by piotrwalczak1** [reply]
I will explain this in another example:
I have:
```
resources:
  - ${file(iam/resources/default-role.yml)}
  - ${file(iam/resources/tables.yml)}
```
I would like my tables file not to include each and every table definitions but have this in a seperate file, so tables.yml:

```
Resources:
  - ${file(users-table.yml)}
  - ${file(cars-table.yml)}
```

Where i.e. users-table.yml:

```
UsersTableDynamo:
  Type: "AWS::DynamoDB:Table"
   ...
```

But when try doing so it does not work:
 The CloudFormation template is invalid: Template format error: Any Resources member must be an object.
 
Is this possible ?

**Reply by pgrzesik** [reply]
Can you post a small repo with a minimal reproducible example? It's quite hard to parse such examples in head and it will probably become much easier to comprehend with concrete example that I can play with. I'm guessing it's the issue of the final structure that ends up in `resources` in your case
